pkg/util/grpcutil/grpc_util.go

```diff
@@ -31,13 +31,13 @@ import (
 
 type localRequestKey struct{}
 
-// NewLocalRequestContext takes an existing context which may have the Peer field set,
-// and returns a Context that can be used for local (in-process) request.
+// NewLocalRequestContext takes an existing context and returns a context that can
```

Comment:
s/takes an existing context // (this is already in the signature)

also s/context/Context/ since you're describing the type.